tf/core/providers.tf

```diff
@@ -4,11 +4,11 @@ terraform {
   required_providers {
     hcloud = {
       source  = "hetznercloud/hcloud"
-      version = "= 1.51.0"
+      version = "= 1.50.1"
```

Comment:
We have to downgrade this provider because it was from pull from unstable which was slightly ahead of the the the new stable.